package.json

```diff
@@ -25,7 +25,8 @@
     "postinstall": "node scripts/npm/postinstall",
     "release": "./scripts/travis/release.sh",
     "build": "gulp build",
-    "test": "gulp test && node test/accessibility/wcag.js"
+    "test": "gulp test && node test/accessibility/wcag.js",
```

Comment:
Sure, mind opening an issue with details so we can address it after the atomic work is completed?